lua/nvim-treesitter/query.lua

```diff
@@ -8,7 +8,7 @@ local M = {}
 
 local query_cache = caching.create_buffer_cache()
 
-M.built_in_query_groups = {'highlights', 'locals', 'textobjects', 'folds'}
+M.built_in_query_groups = {'highlights', 'locals', 'textobjects', 'folds', 'indents'}
```

Comment:
Didn't I remove textobject? Apparently not.

Reply:
would may be nice if it would indent the second line of a multi-line statement in Lua

Reply:
what do you mean by multi-line statement ?
this is indenting based on root level. I might need to implement more advanced indenting for multi line statement if you mean something like:
```lua
local a = some_value or
                some_other_value
```

Reply:
I mean examples like the line you cited. Basically, the second line of all nodes that end in `_statement` could be indented somehow.

Reply:
i still need to optimize this. Computing queries each time is not needed. (see `gg=G` on a file is quite slow, because it will requery for each line in the buffer).